db/seeds.rb

```diff
@@ -712,3 +712,23 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   )
 end
 
+# ----------------------------------------------------------------------------
+# Transfers
+# ----------------------------------------------------------------------------
+storage_location = random_record_for_org(pdx_org, StorageLocation)
+stored_inventory_items_sample = storage_location.inventory_items.sample(20)
+distribution = Distribution.create!(storage_location: storage_location,
```

Comment:
Is there a reason we're creating a new distribution here instead of using one of the existing ones?

Reply:
Nope, was just on autopilot- thanks for the catch! I just put in a new commit that grabs the first transfer instead and checked that the seed is still working.